resources/views/admin/addProduct.blade.php

```diff
@@ -0,0 +1,149 @@
+                            'name' => 'priceForSize[Default]',
+                            'value' => old('priceForSize.Default'),
+                        ])
+                        <!-- Product Size and Price Fields -->
+                        <div>
+                            <div id="size-price-options" class="space-y-4">
+                                <div class="flex items-center space-x-4">
+                                    <input type="checkbox" id="same-price-all" class="form-checkbox text-blue-500 h-5 w-5">
+                                    <label for="same-price-all"
+                                        class="ml-2 text-gray-700">{{ __('manage-products/products.custom_sizes_prices_label') }}</label>
+                                </div>
+                                <div id="size-price-inputs" class="hidden">
+                                    <div id="specific-size-prices">
+                                        <label class="block text-gray-700 font-semibold">
+                                            {{ __('manage-products/products.custom_sizes_label') }} </label>
+                                        <div id="custom-size-inputs">
+                                            <!-- Dit is waar de nieuwe invoervelden worden toegevoegd -->
+                                            @php
+                                                $oldCustomSizes = old('custom_sizes') ?? [];
+                                                $oldCustomPrices = old('custom_prices') ?? [];
+                                            @endphp
+                                            @if (count($oldCustomSizes) > 0 && count($oldCustomPrices) > 0)
+                                                @for ($i = 0; $i < count($oldCustomSizes); $i++)
+                                                    <script>
+                                                        addCustomSizeInput('{{ $oldCustomSizes[$i] }}', '{{ $oldCustomPrices[$i] }}');
+                                                    </script>
+                                                @endfor
+                                            @else
+                                                <script>
+                                                    addCustomSizeInput();
+                                                </script>
+                                            @endif
+                                        </div>
+                                        <button onclick="addCustomSizeInput()" type="button"
+                                            class="mt-2 bg-blue-500 text-white px-4 py-2 rounded-md">{{ __('manage-products/products.custom_size_button') }}</button>
+                                    </div>
+                                </div>
+                            </div>
+                            @php
+                                $price_sizeErrorTypes = [
```

Comment:
Dit kan je, denk ik, beter meegeven vanuit een controller, of een enum maken zoals met de UserRoleEnum